### PR TITLE
fix(client): removal of divider lines between buttons

### PR DESCRIPTION
**Pull request by sonali-dudekula**
Issue Number: #2720 

## PR Details

The code has been modified to remove the divider lines between buttons present at Entities > Permissions. This pull request is opened to reflect the above-made code changes in the original application after it gets merged.

## PR Checklist
- [x] Removal of divider lines between buttons.

### CODE REVIEW

**Comment by yuval-hazaz** [comment]
@tupe12334 please validate and merge to next release

File: packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@
       height: var(--default-button-height);
       padding: 0;
       background-color: var(--black20);
-      border: $border-black10;
+      border: 0px solid var(--black20);
```
Comment (tupe12334) [inline packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss:26]: Why did you add the `solid var(--black20)` part?

Comment (sonali-dudekula) [reply packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss:26]: Even after the border has been made to 0px, a very thin line still exists as divider when observed by zooming in. so I have added `solid var(--black20)` to completely disable the divider.

Comment (tupe12334) [reply packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss:26]: Are you sure? I can see that line
Only 0px:
Screen Shot 2022-05-23 at 10 30 18

Regular:
Screen Shot 2022-05-23 at 10 30 52

With `0px solid var(--black20)`
Screen Shot 2022-05-23 at 10 32 00

Comment (sonali-dudekula) [reply packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss:26]: You mean you *cannot see that line with only `0px`?

Comment (sonali-dudekula) [reply packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss:26]: With only `0px`
Screenshot 2022-05-23 134134

Comment (sonali-dudekula) [reply packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss:26]: Please zoom in the above attached image, you can find a very thin line separating all roles and granular

Comment (tupe12334) [reply packages/amplication-design-system/src/components/MultiStateToggle/MultiStateToggle.scss:26]: Ok so we will merge it like that with the solid
